tests/org/tests_get_group.py

```diff
@@ -0,0 +1,48 @@
+from org.models import *
+from tests.AuthAPITestCase import AuthAPITestCase
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from users.models import User
+from org.serializers import CreateOrgSerializer
+from org.custom_model_field import PermissionField as Permissions
+import uuid
+
+
+class EditOrgAPITestCase(AuthAPITestCase):
+    def setUp(self):
+        # Inheriting the base class funtionality
+        super(EditOrgAPITestCase, self).setUp()
+
+        data_org = {
+            "name": 'test',
+            "tagline": 'test',
+        }
+        serializer = CreateOrgSerializer(data=data_org)
+        if serializer.is_valid():
+            self.org, self.admin_group = serializer.save()
+
+    def test_fail_without_auth(self):
+        get_org_api = "/api/org/1/"
+        un_auth_client = APIClient()
+        response = un_auth_client.get(get_org_api)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_fail_with_unknown_org(self):
+        get_org_api = "/api/org/2/"
```

Comment:
Drawing this from @naveennvrgup sir at #110 , 

> Lets bring in the practice of using really large org ids for invalid orgs. So that minor changes like adding another org wont break the test cases.

```suggestion
        get_org_api = "/api/org/12345/"
```
It would be nicer to do it this way.